### PR TITLE
#162474694 Implement read article functionalities 

### PR DESCRIPTION
**Pull request by TheoOkafor**
#### What does this PR do?
Implement read article functionalities

#### Description of Task to be completed?
- Add the follow user functionality
- Add the unfollow user functionality
- Add the like and love count
- Add like and love article functionality

#### How should this be manually tested?
On the command line;
- git clone this repo
- Run `npm start` from the `haven-ah-frontend` directory
- Visit http://localhost:5001/articles/:slug (where `slug` is the is the slug of the article you want to view.

#### What are the relevant pivotal tracker stories?
[#162474694](https://www.pivotaltracker.com/story/show/162474694)

#### Screenshots (if appropriate)>
screen shot 2018-12-11 at 7 19 23 pm

### CODE REVIEW

**Comment by victor-nweze-deel** [comment]
This is good. Nice work @TheoOkafor

File: src/components/containers/followButton/followButton.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  }
+}
+
+const mapStateToProps = state => ({
+  article: state.articleThread.article,
+  following: state.following.payload,
+});
+
+FollowButton.propTypes = {
+  article: PropTypes.object,
```
Comment (ucheg6) [inline src/components/containers/followButton/followButton.jsx:50]: No default props for props that are not required?

**Comment by ucheg6** [comment]
Good work @TheoOkafor 👍

File: src/components/containers/followButton/followButton.jsx (hunk 1 above)
Comment (TheoOkafor) [reply src/components/containers/followButton/followButton.jsx:50]: Thank you. I'll fix that.

**Review by seyi-adeleke — APPROVED** [review]

**Pull request merged**